test/test-dtutils.lua

```diff
@@ -0,0 +1,116 @@
+--[[
+
+    test-dtutils.lua - the start of a testing framework idea....
+
+    copyright (c) 2016 Bill Ferguson
+
+    TODO: Think this out a little better
```

Comment:
ideally i'd prefer you downloading it from luarocks, maybe with a dtutils.luarocks sublib :P